src/components/FormRegister.vue

```diff
@@ -0,0 +1,221 @@
+<script lang="ts">
+/**
+ * FormRegister Component
+ *
+ * The `FormRegister` component renders registration form.
+ *
+ * @description * Use this component to render registration form. Including
+ * third party registrators and the link to registration of coordinators.
+ *
+ * Note: This component is commonly used in `LoginPage`.
+ *
+ * @events
+ * - `formSubmit`: Emitted on form submit.
+ *
+ * @slots
+ *
+ * @components
+ *
+ * @example
+ * <FormRegister />
+ *
+ * @see [Figma Design](https://www.figma.com/file/L8dVREySVXxh3X12TcFDdR/Do-pr%C3%A1ce-na-kole?type=design&node-id=6356%3A25412&mode=dev)
+ */
+
+import { defineComponent, ref, reactive } from 'vue';
+
+export default defineComponent({
+  name: 'FormRegister',
+  emits: ['formSubmit'],
+  setup() {
+    const formRegister = reactive({
+      email: '',
+      password: '',
+      passwordConfirm: '',
+    });
+
+    const isPassword = ref(true);
+    const isPasswordConfirm = ref(true);
+
+    const isValid = (val: string): boolean => val?.length > 0;
+
+    const isEmail = (value: string): boolean => {
```

Comment:
Same as my previous comment https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/111/files#r1384364179